spreadsheets/sort_rank/landuse.csv

```diff
@@ -1,4 +1,5 @@
 kind,geometrytype,sort_rank
+embankment,*,274
```

Comment:
Seems like this should go below power_line, more just under fence?

Reply:
OK. Fixed in ffe23bd.